crates/wasi-http/src/types.rs

```diff
@@ -117,7 +120,18 @@ pub trait WasiHttpView: IoView {
         request: hyper::Request<HyperOutgoingBody>,
         config: OutgoingRequestConfig,
     ) -> crate::HttpResult<HostFutureIncomingResponse> {
-        Ok(default_send_request(request, config))
+        #[cfg(feature = "default-send-request")]
+        {
+            Ok(default_send_request(request, config))
+        }
+        #[cfg(not(feature = "default-send-request"))]
+        {
+            let _ = (request, config);
+            Err(crate::bindings::http::types::ErrorCode::InternalError(Some(
+                "default-send-request feature disabled".to_string(),
+            ))
+            .into())
+        }
```

Comment:
Personally I don't feel this is the best way to model the behavior here. Could the crate feature be used to control whether this is is a provided function or not? For example without the crate feature this would be an empty function meaning that callers would be required to implement it. With the crate feature though it would be a provided function meaning it has a default implementation.

Reply:
Thanks, missed a `no-default-features` in `wasmtime-cli` otherwise.